packages/console/src/components/Dropdown/use-position.ts

```diff
@@ -0,0 +1,43 @@
+import { RefObject, useCallback, useLayoutEffect, useState } from 'react';
+
+type Position = {
+  left: number;
+  top: number;
+  width: number;
```

Comment:
Do we need `width` in position?

Reply:
YES, some times we need the popup layer's width equals to the anchor's width, "select" is an example.

Reply:
ok, but can we move this `width` logic to that specific `select` component?

Reply:
updated